javascript/chronometer.js

```diff
@@ -1,39 +1,61 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null
   }
 
-  start(callback) {
-    // ... your code goes here
+  start() {
+    this.intervalId = setInterval(() => {
+      console.log(this.currentTime);
+      this.currentTime += 1;
+    }, 1000);
   }
 
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime / 60);
   }
 
   getSeconds() {
-    // ... your code goes here
+    return this.currentTime % 60;
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    if (value < 10) {
+      return ('0' + value.toString());
+    }
+    if (value >= 10) {
```

Comment:
nesse caso nem ha necessidade do segundo if... sempre que o numero for maior que 10, ele vai cair nesse segundo return :)